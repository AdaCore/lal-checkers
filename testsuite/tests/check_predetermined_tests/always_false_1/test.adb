procedure Test is
   y : Integer;
begin
   if y <= 0 then
      y := 40;
   end if;

   if y < 0 then
      y := 50;
   end if;
end Ex1;
