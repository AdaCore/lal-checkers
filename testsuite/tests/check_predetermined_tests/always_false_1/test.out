{
  "always": [
    {
      "kind": false,
      "traces": [
        [
          "assign0",
          "assume0",
          "assume1",
          "assume2",
          "read0",
          "split_join0",
          "start0"
        ]
      ]
    }
  ],
  "le_t_eq_v": [
    {
      "kind": false,
      "traces": [
        [
          "assign0",
          "assume0",
          "assume2",
          "read0",
          "split_join0",
          "start0"
        ],
        [
          "assume1",
          "assume2",
          "read0",
          "split_join0",
          "start0"
        ]
      ]
    }
  ]
}